infra/app-flask/app-config/env-config/environment_variables.tf

```diff
@@ -0,0 +1,34 @@
+locals {
+  # Map from environment variable name to environment variable value
+  # This is a map rather than a list so that variables can be easily
+  # overridden per environment using terraform's `merge` function
+  default_extra_environment_variables = {
+    # Example environment variables
+    # WORKER_THREADS_COUNT    = 4
+    # LOG_LEVEL               = "info"
+    # DB_CONNECTION_POOL_SIZE = 5
+  }
+
+  # Configuration for secrets
+  # List of configurations for defining environment variables that pull from SSM parameter
+  # store. Configurations are of the format
+  # {
+  #   ENV_VAR_NAME = {
+  #     manage_method     = "generated" # or "manual" for a secret that was created and stored in SSM manually
+  #     secret_store_name = "/ssm/param/name"
+  #   }
+  # }
+  secrets = {
```

Comment:
We need to add a secret for the API auth, like in `platform-test-flask`: https://github.com/navapbc/platform-test-flask/blob/a33ecb3355d5a4729dfd12a1723095d32475e2d1/infra/app/app-config/env-config/environment_variables.tf#L22-L25